ledger-tool/src/main.rs

```diff
@@ -1817,11 +1821,11 @@ fn main() {
                         working_bank.print_accounts_stats();
```

Comment:
Sidenote: what do you think about updating the help text for `--print-accounts-stats` to note that stats are printed as `info`-level logs?

Reply:
I noticed that too - starting digging into the function to see if the function created intermediate objects that implemented `Serialize` but was then surprised to see it just does stuff via log.

Yeah, I think we could update the help text. Maybe separate from this PR ? I think ideally, we could clean up those functions to return types that could implement `Display` or `Serialize`. That being said, I'm not sure I have ever used this flag so probably a back-burner item

Reply:
Oh yes, separate PR for sure